megamek/src/megamek/server/Server.java

```diff
@@ -21141,6 +21141,25 @@ private void resolveShutdownCrashes() {
             critBonus = Math.min((damage_orig - 1) / 5, 4);
         }
 
+        // Find out if Human TRO plays a part it crit bonus
+        Entity ae = game.getEntity(hit.getAttackerId());
+        if (ae != null) {
+            if (!ae.getCrew().getOptions().stringOption(OptionsConstants.MISC_HUMAN_TRO).isEmpty() && !areaSatArty) {
```

Comment:
This is purely stylistic, but I would combine the null check and the human TRO check into one conditional.  This reduces the level of indentation and is safe because of short-circuiting.

Also, the check for unit types should be else-if's.  You can't have a Mech-Aero, or Aero-Tank, so only one condition ever applies.